src/BuildScriptGenerator/Node/NodeConstants.cs

```diff
@@ -23,6 +23,8 @@ internal static class NodeConstants
         internal const string AllNodeModulesDirName = "__oryx_all_node_modules";
         internal const string ProdNodeModulesDirName = "__oryx_prod_node_modules";
         internal const string NodeModulesDirName = "node_modules";
-        internal const string NodeModulesZippedFileName = "node_modules.tar.gz";
+        internal const string NodeModulesZippedFileName = "node_modules.zip";
+        internal const string NodeModulesTarGzFileName = "node_modules.tar.gz";
+        internal const string NodeModulesFileBuildProperty = "compressedNodeModulesFile";
```

Comment:
The naming could be a bit confusing - it ends with `BuildProperty`, but it isn't used as a `[BuildProperty(...)]` annotation for the `NodePlatform` class. I guess we shouldn't have two separate sets of definitions that are both called Build Properties

Reply:
Good point., although the toml file is only storing the build properties that were derived from what was passed in as build properties...